### PR TITLE
Matrix_intersection in java

### PR DESCRIPTION
**Pull request by raksha009**
issue no #1628 

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Addition of new code
- 

#### Languages Used:
- Java
- 

#### Files Added:
- Matrix_intersection.java
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by raksha009** [comment]
plz review my PR @singh-shreya6 @asha15 @raghav-dalmia

File: Matrix_Operations/Java/Matrix_intersection.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            System.out.println(" Enter elements of 1st matrix ");
+            for (int i = 0 ; i < rows_first; i++) {
+                for (int j = 0 ; j< columns_first; j++) {
+                    matrix1[i][j] = s.nextInt();
+                }
+            }
+
+            System.out.println(" Enter elements of 2nd matrix ");
+            for (int i = 0 ; i < rows_first; i++) {
+                for (int j = 0 ; j < columns_first; j++) {
```
Comment (asha15) [inline Matrix_Operations/Java/Matrix_intersection.java:55]: please remove the space before ;

**Comment by raksha009** [comment]
Done @asha15

**Comment by asha15** [comment]
Good. have you create PR for all the matrix operations in java?

**Comment by raksha009** [comment]
@asha15  maam, I will make all the PRs by tomorrow max.

**Comment by raksha009** [comment]
@asha15 maam, I have made all the PRs. Plz Review my PR.
Thanks

**Comment by raksha009** [comment]
I have made all the PRs. Plz Review my PR @asha15 maam
Thanks

**Comment by raksha009** [comment]
Sorry, I tried squashing the commits for this PR but ended up messing my master branch. I was able to correct the error in my master branch but I am not able to squash the commits for this PR.

**Comment by singh-shreya6** [comment]
Close this PR @raksha009. Open a new one with proper changes.

**Comment by raksha009** [comment]
okk, will do it right now. Thanks @singh-shreya6 maam

**Comment by raksha009** [comment]
Done maam, I opened a same Pr for the same #2877. pl review @singh-shreya6

**Pull request closed without merging**